scripts/data_utils.py

```diff
@@ -429,6 +434,28 @@ def merge_chunks_serially(chunked_content_list: List[str], num_tokens: int) -> G
         yield current_chunk, total_size
 
 
+def get_embedding(text):

```

Comment:
Just a thing to consider. No need to address in this PR. I think we should be able to use open ai SDK for embedding instead of using our own home-grown client